Exiled.API/Features/Roles/FpcRole.cs

```diff
@@ -110,6 +111,16 @@ public PlayerMovementState MoveState
         /// </summary>
         public bool IsInDarkness => FirstPersonController.InDarkness;
 
+        /// <summary>
+        /// Gets a value indicating whether or not this role is protected by a hume shield.
+        /// </summary>
+        public bool IsHumeShieldedRole => this is IHumeShieldRole;
+
+        /// <summary>
+        /// Gets a value indicating the role's maximum Hume Shield. This will always be <c>0</c> if <see cref="IsHumeShieldedRole"/> is <see langword="false"/>.
+        /// </summary>
+        public float MaxHumeShield => !IsHumeShieldedRole ? 0f : (Base as IHumeShieldedRole).HumeShieldModule.HsMax;
```

Comment:
```suggestion
```
It's already included in the `HumeShieldBase` class, you can wrap it inside hume shield roles or just leave it as it is.